app/views/observations/show/_observation_details.html.erb

```diff
@@ -4,12 +4,12 @@
 
 <%= observation_details_when_where_who(obs: obs) %>
 
-<%= if @user
-  tag.div(class: "obs-projects", id: "observation_projects") do
-    obs.projects.each do |project|
-      tag.p("#{:PROJECT.t}: #{link_to_object(project)}")
-    end
-  end
+<% if @user
+  obs.projects.each do |project| %>
+    <%= "#{:PROJECT.t}: " %>
+    <%= link_to_object(project) %>
+    <br/>
+  <% end
```

Comment:
Yep, I have no idea why we were showing projects only if `@user` — i tried to faithfully reproduce every little thing I didn't understand when re-writing these! Did not always succeed.

I'd definitely like to keep the surrounding div "#observation_projects", and now that I look at it, it's a bit weird to have the "PROJECT:" title repeat on every line when there are multiple projects.

@JoeCohen the way to go is either to `concat` each line that should accumulate (the only option in a loop) or `[array, of, tags].safe_join`

How about this
```suggestion
<%= tag.div(class: "obs-projects", id: "observation_projects") do
  [tag.span("#{:PROJECTS.t}:")), tag.br].safe_join
  obs.projects.each do |project|
    concat(link_to_object(project))
    concat(tag.br)
  end
end %>
```

Reply:
Approved with this change